lib/server/routes/health.js

```diff
@@ -23,7 +27,7 @@ HealthRouter.prototype.health = function(req, res) {
 
 HealthRouter.prototype._definitions = function() {
   return [
-    ['GET', '/health', this.health]
+    ['GET', '/health', this.getLimiter(limiter(1000)), this.health]
```

Comment:
My thought was since  had that one IP that was hammering our frames endpoint, we don't want our health endpoint to be vulnerable to that, either. Even though it's fairly lightweight, I think minimizing our surface area to abuse is probably a good idea.